docs/source/index.rst

```diff
@@ -97,10 +97,19 @@ The example below shows how to view all dataset-level issues in one line of code
    health_summary(labels, pred_probs, class_names=class_names)
 
 
+Contributing 
+------------
+
+As cleanlab is an open-source project, we welcome contributions from the community.
+
+Please see our `contributing guidelines <https://github.com/cleanlab/cleanlab/blob/master/CONTRIBUTING.md>`_ for more information.
+
+
 .. toctree::
    :hidden:
 
    Quickstart <self>
+   How to contribute <https://github.com/cleanlab/cleanlab/blob/master/CONTRIBUTING.md>
```

Comment:
I'd vote to move this further below under "Guides" section. 
Quickstart & Tutorials should be the first thing people see IMO (especially frequent visitors of the docs), not how to contribute

Reply:
The mutli line formatting and small width of the left panel makes the tutorials take up too much space pushing everything pretty far down and requiring a lot more scrolling per content than is ideal.

Fine to move further down I guess but this probably requires another pr to allow a lot more content to be viewed quickly on the left panel then currently

Reply:
moved it for now, reformatting docs layout will be a bigger effort